### ISSUE TITLE
Setting of backup flags not possible

### ISSUE THREAD

**Issue by berti34** [garethgeorge/backrest#344]
I think it is a bug: when I try to place a "Backup flag" like "--insecure-tls" in the settings of a plan or repository the submit is not possible.
It seems that the field doesn't accept --... (double dash) parameters.

![image](https://github.com/garethgeorge/backrest/assets/47418901/66e0ceac-ecc1-4a9e-9eb1-a7c5631f4bc6)

Greetings 
Berthold

**Comment by garethgeorge** [comment]
Hmm, I'm not able to reproduce this. What backrest version are you running? As an aside in your screenshot that looks a bit like an em-dash `—` rather than a double dash `--`

**Comment by berti34** [comment]
You are right, I cannot reproduce it on my desktop browsers too, but on mobile browsers on my iPhone. The mobile browser changes the double dash to em-dash automatically. That's the problem.
Sorry for confusing - and my english.
Is it possible to ignore/stop the autocorrection in the html form?

**Comment by berti34** [comment]
Maybe this helps a bit:
https://weblog.west-wind.com/posts/2015/Jun/15/Turn-off-HTML-Input-Auto-Fixups-for-Mobile-Devices
https://www.tutorialspoint.com/how-to-disable-browser-autocomplete-on-the-web-form-field-input-tag

**Comment by willyouvba** [comment]
This seems to be iOS related: https://support.apple.com/en-au/guide/numbers-iphone/tane7f9690fb/ios
I have the same issue on any site.

**Comment by berti34** [comment]
In version 1.7.0 the behaviour keeps going wrong.

Maybe backrest can be able to accept the wrong character and replace it internally before saving the configuration?
With a warning, not as blocking error?
